### DISCUSSION TITLE
API based scanning

### DISCUSSION THREAD

**Discussion by detronetdip** [kubescape/kubescape#840] [Ideas]
I was thinking of implementing a REST API endpoint where we can send our YAML files and it will be returning the results.  I'm very newbie to opensource contribution and need further suggession regarding my idea.

**Comment by dwertent** [comment]
You should first think of the use case and why it is needed.
Once it is clear to you, you should open an issue with a detailed feature request :)

**Reply by detronetdip** [reply]
@dwertent Thank you
